docs/erp_steuerung_durch_le.adoc

```diff
@@ -315,7 +315,11 @@ Voraussetzung für die Verwendung des KIM-Dienstes ist, das alle beteiligten Par
 
 ==== KIM-Nachricht generieren und Empfänger ermitteln
 Im ersten Schritt wird eine Nachricht im Primärsystem erstellt. Der verordnende Leistungserbringer verfasst einen Nachrichtentext und kann wählen, ob eine Zustellbestätigung erfolgen soll. Das E-Rezept Token wird automatisch in die Nachricht eingefügt. +
-Die Nachricht kann nur an Empfänger versendet werden, für die ein Eintrag im Verzeichnisdienst (inklusive KIM Adresse) der TI vorhanden ist. Der KIM-Header "To" muss mit einer Email-Adresse aus dem Verzeichnisdienst befüllt werden. Das Primärsystem kann hierzu eine Abfrage der Empfänger-Adressen durchführen und agiert dabei als LDAP-Client gegenüber dem LDAP-Server (Verzeichnisdienst). Der Konnektor dient dabei als LDAP-Proxy.
+Die Nachricht kann nur an Empfänger versendet werden, für die ein Eintrag im Verzeichnisdienst (inklusive KIM Adresse) der TI vorhanden ist.
+
+Der KIM-Header "To" muss mit einer Email-Adresse aus dem Verzeichnisdienst befüllt werden. Das Primärsystem kann hierzu eine Abfrage der Empfänger-Adressen durchführen und agiert dabei als LDAP-Client gegenüber dem LDAP-Server (Verzeichnisdienst). Der Konnektor dient dabei als LDAP-Proxy.
+
+Der KIM-Header "KIM-Encounter-Id" ist für die Nachvollziehbarkeit des Nachrichtenaustauschs ebenfalls zu setzen, siehe link:https://github.com/gematik/api-kim/blob/develop/docs/Primaersystem.adoc#43-nachrichten-versenden[API-KIM: Nachrichten Versenden].
```

Comment:
Ich bin dafür, in unserer API-Beschreibung bereits mehr Informationen bereitzustellen und die Motivation zur Verwendung dieses Identifikators zu erhöhen. Z.b. so:

> Der KIM-Header "KIM-Encounter-Id" dient dazu, Nachrichten zu gruppieren, insbesondere im Zusammenhang mit einem Behandlungskontext. Diese ID sollte zu Beginn einer Nachrichtenkonversation festgelegt und bei der Beantwortung von empfangenen Nachrichten beibehalten werden. Weitere Informationen finden Sie in der Dokumentation unter dem folgenden Link: [API-KIM: Nachrichten Versenden](https://github.com/gematik/api-kim/blob/develop/docs/Primaersystem.adoc#43-nachrichten-versenden).